<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectSerialPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GenerateValid">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gen Valid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GenerateInvalid">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gen Invalid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConfigureSerialPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>216</width>
      <height>144</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_SendRevI">
       <property name="text">
        <string>Send RI</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dataRI"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_SendFI">
       <property name="text">
        <string>Send FI</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QLineEdit" name="dataFI"/>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QPushButton" name="pushButton_SendRV">
       <property name="text">
        <string>Send RV</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="dataRV"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_SendFV">
       <property name="text">
        <string>Send FV</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="dataFV"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_SendData">
       <property name="text">
        <string>Send Data</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="dataFull"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
